### DISCUSSION TITLE
What is the usefulness of pool_size: x in main_db inside postal.yml ?

### DISCUSSION THREAD

**Discussion by bukunmilab** [postalserver/postal#3169] [Help with using Postal]
So what does the pool_size do and what would be an ideal value for the pool _size of main_db for postal running in docker in a VPS spec 4 CPU cores, 12GB RAM, 50GB NVme SSD?

```
main_db:
  pool_size: 200
```

**Comment by willpower232** [comment]
The setting is described as `The maximum size of the MariaDB connection pool` so it could be useful for configuring the underlying framework.

The default value is 5 so unless you're running hundreds of instances of Postal on a powerful server, you can probably go for a lower number.

**Reply by bukunmilab** [reply]
I run only Postal on the server.
What value would you recommend for my server spec?
